src/command.rs

```diff
@@ -195,6 +197,51 @@ pub async fn package(directory: impl AsRef<Path>, dry_run: bool) -> miette::Resu
         ))
 }
 
+#[cfg(feature = "git")]
```

Comment:
In this function, I tried to emulate the old behavior that would not `bail` if the command failed to run altogether.